server/Makefile

```
@@ -153,6 +153,7 @@ PLUGIN_PACKAGES += mattermost-plugin-nps-v1.3.3
 PLUGIN_PACKAGES += mattermost-plugin-servicenow-v2.3.4
 PLUGIN_PACKAGES += mattermost-plugin-zoom-v1.8.0
 PLUGIN_PACKAGES += mattermost-plugin-ai-v0.8.1
+PLUGIN_PACKAGES += mattermost-plugin-focalboard-v8.0.0
```

@Rajat-Dabade I think this needs to be the name of the repo so it can find the package. We need to change it to:
```suggestion
PLUGIN_PACKAGES += focalboard-v8.0.0
```